apps/comps/components/clock/index.tsx

```diff
@@ -5,25 +5,50 @@ import { cn } from "@recallnet/ui2/lib/utils";
 interface CountdownClockProps {
   targetDate: Date;
   className?: string;
+  showDuration?: boolean;
 }
 
 export const CountdownClock: React.FC<CountdownClockProps> = ({
   targetDate,
   className,
+  showDuration,
 }) => {
   const [timeLeft, setTimeLeft] = useState({
     hours: 0,
     minutes: 0,
     seconds: 0,
   });
+  const [duration, setDuration] = useState("");
   const [isCountdownFinished, setIsCountdownFinished] = useState(false);
 
+  const pluralize = (amount: number, word: string) =>
+    `${word}${amount > 1 ? "s" : ""}`;
   useEffect(() => {
     const updateCountdown = () => {
       const now = new Date();
       const difference = targetDate.getTime() - now.getTime();
 
       if (difference > 0) {
+        const days = Math.floor(difference / (1000 * 60 * 60 * 24));
+        if (showDuration === true) {
```

Comment:
To match the design I added a `showDuration` prop that will show more human readable info until the time left is less than 48 hours.

Reply:
Good call, I'll change this